<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveFromMaintenance</class>
 <widget class="QDialog" name="RemoveFromMaintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="startDateReport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start date: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="removeList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="staffNameReport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Staff:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deadlineReport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Deadline: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
